src/data/finset/powerset.lean

```diff
@@ -10,9 +10,9 @@ import data.finset.lattice
 -/
 
 namespace finset
-open multiset
+open function multiset
 
-variables {α : Type*}
+variables {α : Type*} {s t : finset α}
```

Comment:
Not convinced these are worth it for a single lemma; can you just put them on `powerset_inj`?

Reply:
The rest of the section could use them, and indeed `powerset_eq_singleton_empty` uses `s`.